examples/Server/Startup.cs

```diff
@@ -34,7 +32,7 @@ public void ConfigureServices(IServiceCollection services)
         }
 
         // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
-        public void Configure(IApplicationBuilder app, IHostingEnvironment env)
```

Comment:
`IHostingEnvironment` will be deprecated in favour of `IWebHostEnvironment` but https://github.com/aspnet/AspNetCore/pull/7725 isn't merged yet. I didn't want to wait until the changes go in and a new SDK build so I opted to just remove the argument since we weren't using it anyway.